simplified-ui-splash/src/main/java/org/nypl/simplified/ui/splash/MailtoWebViewClient.java

```diff
@@ -0,0 +1,52 @@
+package org.nypl.simplified.ui.splash;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.MailTo;
+import android.net.Uri;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import java.lang.ref.WeakReference;
+
+/**
+ * Used to handle mailto: links in the eula.html
+ */
+public class MailtoWebViewClient extends WebViewClient {
+    private final WeakReference<Activity> mActivityRef;
+
+    public MailtoWebViewClient(Activity activity) {
+        mActivityRef = new WeakReference<>(activity);
+    }
+
+    @Override
+    public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        if (url.startsWith(MailTo.MAILTO_SCHEME)) {
+            final Activity activity = mActivityRef.get();
+            if (activity != null) {
+                try {
+                    MailTo mt = MailTo.parse(url);
+                    Intent i = newEmailIntent(activity, mt.getTo(), mt.getSubject());
+                    if (i.resolveActivity(activity.getPackageManager()) != null) {
+                        activity.startActivity(i);
+                    }
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+                return true;
+            }
+        } else {
+            view.loadUrl(url);
```

Comment:
I tried to leave this comment earlier, but apparently it didn't take for some reason.

The docs specifically warn about calling `loadUrl` and then returning `true` from this method. Instead, you should just return `false` and let the WebView continue loading as normal. Like this:

```java
if (url.startsWith(MailTo.MAILTO_SCHEME)) {
    // ...
    return true;
}
return false;
```

> Note: Do not call WebView#loadUrl(String) with the same URL and then return true. This unnecessarily cancels the current load and starts a new load with the same URL. The correct way to continue loading a given URL is to simply return false, without calling WebView#loadUrl(String).

https://developer.android.com/reference/android/webkit/WebViewClient#shouldOverrideUrlLoading(android.webkit.WebView,%20java.lang.String)